robowflex_movegroup/src/services.cpp

```diff
@@ -97,11 +98,18 @@ void MoveGroupHelper::setResultCallback(const ResultCallback &callback)
     callback_ = callback;
 }
 
-bool MoveGroupHelper::executeTrajectory(const robot_trajectory::RobotTrajectory &path)
+bool MoveGroupHelper::executeTrajectory(robot_trajectory::RobotTrajectory &path)
 {
     if (!eac_.isServerConnected())
         return false;
 
+    // Check if a Trajectory is time parametirized
+    if (path.getWayPointDurationFromStart(path.getWayPointCount()) < 1e-3)
```

Comment:
I don't like this for checking if the path is time parameterized or not. This would spuriously compute the parameterization for short paths, like going from a state to the same state. I think a better check would be to see if all the waypoints in the path have no velocity, acceleration, or effort specified instead.